pilot/pkg/model/sidecar_test.go

```diff
@@ -2095,73 +2096,93 @@ func TestIstioEgressListenerWrapper(t *testing.T) {
 
 	tests := []struct {
 		name          string
-		listenerHosts map[string][]host.Name
+		listenerHosts map[string]hostClassification
 		services      []*Service
 		expected      []*Service
 		namespace     string
 	}{
 		{
-			name:          "*/* imports only those in a",
-			listenerHosts: map[string][]host.Name{wildcardNamespace: {wildcardService}},
-			services:      allServices,
-			expected:      []*Service{serviceA8000, serviceA9000, serviceAalt},
-			namespace:     "a",
+			name: "*/* imports only those in a",
+			listenerHosts: map[string]hostClassification{
```

Comment:
Only `listenerHosts` has been modified here, and other fields have not been modified, it is format diff.